kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/ProcurementCardLevel3AddItem.java

```diff
@@ -0,0 +1,144 @@
+package edu.arizona.kfs.fp.businessobject;
+
+import java.math.BigDecimal;
+import java.util.LinkedHashMap;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+public class ProcurementCardLevel3AddItem extends PersistableBusinessObjectBase {
+
+	private String documentNumber;
+	private Integer financialDocumentTransactionLineNumber;
+	private Integer sequenceNumber;
+	private String itemCommodityCode;
+	private String itemProductCode;
+	private String itemDescription;
+	private BigDecimal itemQuantity;
```

Comment:
I am not sure why this property is specified as BigDecimal in the original KFS 3.0 implementation. The JPMC specs have it as Numeric, max length 12, right justified and zero padded.